Vagrantfile

```diff
@@ -0,0 +1,15 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+$script = <<SCRIPT
+    dnf install -y python-flask python-flask-wtf python-flask-openid python-wtforms python-openid python-docutils python-dateutil python-markupsafe python-bunch python-straight-plugin python-setuptools python-sqlalchemy fedmsg
+    cd /opt/anitya/src/ && python createdb.py
+SCRIPT
+
+Vagrant.configure("2") do |config|
+  config.vm.box = "box-cutter/fedora23"
```

Comment:
@puiterwijk, there isn't a fedora24 Vagrant image yet from a reputable source. I'm waiting on this PR to go through:
https://github.com/boxcutter/fedora/pull/20